content/synchronization_model.tex

```diff
@@ -19,9 +19,9 @@
 \begin{tabular}{p{0.2\textwidth} | p{0.7\textwidth}}
 {}
 &
-{ Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
+Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
 used when computation on the local \ac{PE} cannot proceed without the value that
-the remote \ac{PE} is to update.} \tabularnewline
+the remote \ac{PE} is to update. \tabularnewline
```

Comment:
Why are these curly braces removed?  This edit appears to be unrelated to this proposal.

Reply:
Why were the braces removed?

Reply:
It was a part of various additions and deletions. I can add back. It seems to have impact

Reply:
it seems to have *no* impact.

Reply:
Yeah, the curly braces are a bit odd.  If you verified that the PDF is unchanged, then I'm ok with the change.

Reply:
The pdf looks fine. Also, some of the other items in the table doesn’t have curly braces.